modules/ROOT/pages/dataweave-logging-configuration.adoc

```diff
@@ -0,0 +1,118 @@
+= DataWeave Logging Configuration
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The logging configuration feature provides granular control over DataWeave logging, which improves debugging and monitoring of data transformations.
+
+Starting with DataWeave 2.10, you can configure logging to control the verbosity of log messages using standard log levels: `Debug`, `Info`, `Warn`, and `Error`. You can filter log messages by severity and scope, which makes it easier to troubleshoot and monitor your DataWeave code.
+
+== Configuration Location
+
+Define the logging configuration in a DataWeave language file named `log-config.dwl`. Place this file in the classpath at the root level.
+
+Use the `DataWeaveScriptingEngine` API to customize the name, location, and format of the configuration file by either implementing the `LoggingConfigurationSourceProvider` interface or by using the helper class `DefaultLoggingConfigurationSourceProvider`.
```

Comment:
This section regarding `DataWeaveScriptingEngine` API does not apply to DataWeave in Mule Runtime, not sure what's the best way to clarify it